rosetta-ide/src/main/java/com/regnosys/rosetta/ide/server/RosettaLanguageServerImpl.java

```diff
@@ -54,6 +78,14 @@ protected ServerCapabilities createServerCapabilities(InitializeParams params) {
 			serverCapabilities.setInlayHintProvider(inlayHintRegistrationOptions);
 		}
 		
+		if (resourceServiceProvider.get(ICodeActionService2.class) != null) {
+            CodeActionOptions codeActionProvider = new CodeActionOptions();
+            codeActionProvider.setResolveProvider(true);
+            codeActionProvider.setCodeActionKinds(List.of(CodeActionKind.QuickFix));
```

Comment:
I've noticed there is also a `CodeActionKind.SourceOrganizeImports`. Not sure if it matters, but might be good to have a look at the specfication to see how this is used and whether we wanna use this code action kind instead? (maybe not for the quick fixes, but just for the sorting?)